lib/mix/tasks/phx.gen.ex

```diff
@@ -18,6 +18,33 @@ defmodule Mix.Tasks.Phx.Gen do
   | `phx.gen.live`     | x | x | x |   |   | x |
   | `phx.gen.json`     | x | x | x | x | x |   |
   | `phx.gen.html`     | x | x | x | x | x |   |
+
+
+  ## Customizing generators
+  
+  You can override the default templates used by generators.
+  
+  For example, to customize `phx.gen.live`, you can copy and edit the generator templates.
+  A common pattern is to place them under `priv/templates/phx.gen.live/` in your project:
+  
+  Create the directory for your custom `phx.gen.live` templates:
+  ```console
+  $ mkdir -p priv/templates/phx.gen.live
+  ```
+
+  Copy the default phx.gen.live generator templates into your project so you can customize them:
+
+  ```console
+  $ cp -r deps/phoenix/priv/templates/phx.gen.live/* priv/templates/phx.gen.live/
+  ```
+  Phoenix generators will look for templates in your project's `priv/templates` directory first.
+  If a matching template is found, it will be used instead of the built-in default.
+
+  See: `https://fly.io/phoenix-files/customizing-phoenix-generators/` for a more detailed approach.
+
+  `Note: Generator templates may change between minor Phoenix releases,
+  so custom templates may require updates after upgrading.`
+
```

Comment:
````suggestion
  ## Customizing generators
  
  You can override the default templates used by generators.
  
  For example, to customize `phx.gen.live`, you can copy and edit the generator templates.
  A common pattern is to place them under `priv/templates/phx.gen.live/` in your project:
  
  Create the directory for your custom `phx.gen.live` templates:

  ```console
  $ mkdir -p priv/templates/phx.gen.live
  ```

  Copy the default phx.gen.live generator templates into your project so you can customize them:

  ```console
  $ cp -r deps/phoenix/priv/templates/phx.gen.live/* priv/templates/phx.gen.live/
  ```

  Phoenix generators will look for templates in your project's `priv/templates` directory first.
  If a matching template is found, it will be used instead of the default.

  Note generator templates may change between minor or even patch Phoenix releases,
  so custom templates may require updates after upgrading. Use this mechanism at your
  own risk.
```